.github/workflows/release.yml

```diff
@@ -0,0 +1,35 @@
+# GitHub release for GCP versioning
+name: release
+
+# the release workflow is triggered by each push with new tag
+on:
+  push:
+    tags:
+      - '*'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Create Release
+        uses: ncipollo/release-action@v1
```

Comment:
Like in PR #35, we don't want extra third-party dependencies here, especially for the release process. We would prefer to explore native Github mechanism (if any) for that.

Reply:
Understood. Thanks a lot for the advice. I will be working on that.

Reply:
> Like in PR #35, we don't want extra third-party dependencies here, especially for the release process. We would prefer to explore native Github mechanism (if any) for that.

I have removed the third-party dependencies and committed a new file. It works fine in my forked repository. I added the usage and notes. Please take a look:).